pkg/column-mapping/column.go

```diff
@@ -15,31 +15,59 @@ package column
 
 import (
 	"fmt"
+	"strconv"
 	"sync"
 
 	"github.com/juju/errors"
 	"github.com/pingcap/tidb-tools/pkg/table-rule-selector"
 )
 
+var (
+	// for partition ID, ref definition of partitionID
+	instanceIDBitSize       = 4
+	schemaIDBitSize         = 7
+	tableIDBitSize          = 8
+	maxOriginID       int64 = 17592186044416
+)
+
+// SetPartitionRule sets bit size of schema ID and table ID
+func SetPartitionRule(instanceIDSize, schemaIDSize, tableIDSize int) {
+	instanceIDBitSize = instanceIDSize
+	schemaIDBitSize = schemaIDSize
+	tableIDBitSize = tableIDSize
+	maxOriginID = 1 << uint(64-instanceIDSize-schemaIDSize-tableIDSize-1)
```

Comment:
do we need to check the total size of `instanceIDSize`, `schemaIDBitSize`, `tableIDBitSize`